Asterix/Example_param_file.ini

```diff
@@ -267,13 +267,22 @@ onbench=True
     name_DM_to_probe_in_PW = 'DM2'
 
     #Amplitude of PW probes (in nm)
-    amplitudePW = 34
+    amplitudePW = 134
 
     #Actuators used for PW (DM in pupil plane)
-    posprobes = 466,498
+    posprobes = 592,595,736# 592,595,736#297,298,266 # put 592,595,736 for gaussian (1.4) roman with 48x48, 592,592,592 for sinc ROMAN (48x48)
+    # for actu 48x48: 592,593,640
+    #for gaussian 48x48: 592,595,736
+    #for sinc 48x48: 592,592,592
     #For DM2
     #posprobes = 392,393, 425
-
+    
+    #Choosing which probe shape to use "actu", "sinc", "gaussian"
+    probes_shape = 'gaussian'
```

Comment:
param probes_shape can be added to the default param file, but is should be "actu"